app/assets/stylesheets/custom.scss

```diff
@@ -229,4 +229,14 @@ a.like_btn {
 .comment-comments {
   margin-left: 10%;
   border-left: 2px solid gainsboro;
+}
+.social {
+  height: 30px;
+}
+#facebook-auth {
+  background-color: #486090 !important;
```

Comment:
please don't use `!important` at all

Reply:
i know that(I tried to rewrite bootstrap li.dropdown classes ). Sorry I forgot to remove that.